build.gradle

```diff
@@ -30,7 +30,6 @@ dependencies {
     runtimeOnly('com.h2database:h2')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
     compileOnly('org.projectlombok:lombok')
-    annotationProcessor('org.projectlombok:lombok')
```

Comment:
Prawdopodobnie przez usunięcie tej linijki wywalają się testy na circleCI. Czemu to chciałas wyrzucić?

Reply:
Kombinowałam w jaki sposób pozbyc się błędu o którym wcześniej pisałam i zasugerowałam się tym, że na jakiejść stronie internetowej zobaczyłam że wprowadzona zależność może byc bez tego. Nie ważne już spowrotem przywróciłam dawny stan w build.gradle